core/src/mindustry/world/blocks/defense/MendProjector.java

```diff
@@ -43,7 +43,7 @@ public MendProjector(String name){
     public boolean outputsItems(){
         return false;
     }
-
+    
```

Comment:
?

Reply:
Dunno why that happened or when I did that.